jigasi.yml

```diff
@@ -12,6 +12,8 @@ services:
             - ${CONFIG}/transcripts:/tmp/transcripts:Z
         environment:
             - ENABLE_AUTH
+            - ENABLE_GCLOUD_TRANSCRIPTION
```

Comment:
This change is not backwards compatible. I'd say we should either repurpose `ENABLE_TRANSCRIPTIONS` to take a boolean (default to gcloud) or the type of transcription.

Alternatively some `TRANSCRIPTION_BACKEND_TYPE` which one could set to `google` or `vosk`